ovs/lib/scheduledtask.py

```diff
@@ -247,8 +252,9 @@ def gather_scrub_work():
         local_storage_router = None
         local_scrub_location = None
         local_vdisks_to_scrub = []
-        result_set = ResultSet([])
+        result_set = {}
```

Comment:
result_set is the list of results from each running task (the guids of scrubbed disks) - no info about storagerouters
storage_router_list is the list of storagerouters
scrub_map is a helper dictionary to keep track of what guids were assigned to each storagerouter

so i don't think they can be merged